src/Sentry.Serilog/Sentry.Serilog.csproj

```diff
@@ -13,8 +13,12 @@
     <ProjectReference Include="../../src/Sentry/Sentry.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="$(TargetFramework) != 'netstandard2.1'">
     <PackageReference Include="Serilog" Version="2.7.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework) == 'netstandard2.1'">
+    <PackageReference Include="Serilog" Version="2.10.0" />
```

Comment:
Serilog 2.10 came up recently but has a ns2.1 target so lets try this.
A fallback would be to target Sentry.Serilog of ns2.0 if one can't bump Serilog to 2.10 for some reason.